internal/capability/operand_install.go

```diff
@@ -118,8 +120,15 @@ func (ca *capAudit) OperandInstall(ctx context.Context) error {
 	}
 	defer file.Close()
 
-	ca.OperandInstallJsonReport(file)
-	ca.OperandTextReport(os.Stdout)
+	if err := ca.OperandInstallJsonReport(file); err != nil {
+		logger.Debugf("Error during the OperatorInstall Report: %w", err)
+		return err
+	}
+
+	if err := ca.OperandTextReport(os.Stdout); err != nil {
+		logger.Debugf("Error during the OperatorInstall Text Report: %w", err)
```

Comment:
Fix typos: "Error during the OperatorInstall Report: %w" to "Error during the OperandInstall"